ui/components/institutional/interactive-replacement-token-notification/interactive-replacement-token-notification.scss

```diff
@@ -0,0 +1,17 @@
+.interactive-replacement-token-notification {
+  gap: 10px;
+  font-size: 12px;
```

Comment:
This is overriding the Text component... Use the Text components props to specify font-size, not css.